### ISSUE TITLE
Edit page for a collection is borked if a fileset chosen as CollectionRepresentative is deleted

### ISSUE THREAD

**Issue by lsat12357** [OregonDigital/OD2#3156]
### Descriptive summary
The edit page for a collection will not load if a fileset that is selected as a CollectionRepresentative is deleted.

### Expected behavior
Check for existence and handle if not found; allow edit page to load; user should be able to remove CollectionRepresentatives in the branding panel.

### Related work

Link to related tickets or prior related work here.

### Accessibility Concerns

Add any information here to indicate any known or suspected accessibility issues for this ticket

**Comment by straleyb** [comment]
Yeah lets do that. I think it was probably because I filled it with an empty value. Its a different problem though so we can mark this as resolved and make a ticket about the box being weird.

**Comment by KevinJonesMeta** [comment]
Great. I can make that ticket late this morning.
